code/items/weapons/TTTWeapon.cs

```diff
@@ -157,6 +160,22 @@ public override void Simulate(Client owner)
                 }
             }
 
+            if (Input.Released(InputButton.View) && AmmoClip > 0) //Todo: Move this to be a bindable key (default Z???)
```

Comment:
This comment can be removed when about to be merged, should make an issue for it on our github. Or add onto the current `ammosystem` issue

Reply:
Keeping this comment open until good to merge.